contrib/sync_namespaces_jobs.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+import argparse
```

Comment:
How about we add a docstring that explains why you'd want to use this script? something about load testing, etc. You can have a longer description here, than what's in the argument parser description.